packages/io/__tests__/io.test.ts

```diff
@@ -331,11 +332,26 @@ describe('rmRF', () => {
     await fs.appendFile(filePath, 'some data')
     await assertExists(filePath)
 
-    const fd = await fs.open(filePath, 'r')
-    await io.rmRF(testPath)
-
-    await assertNotExists(testPath)
+    // for windows, we need to explicitly set an exlcusive lock
+    // or we won't get an EBUSY error. this can be fixed after
+    // https://github.com/libuv/libuv/issues/3267 is resolved
```

Comment:
Can you move this inside the try statement? It's about the EPERM comment. I don't think Node considers its `fs.open` call not creating a lock on Windows a bug.